MaterialLib/MPL/Properties/pIdealGasLaw.h

```diff
@@ -0,0 +1,44 @@
+/**
+ * \author Norbert Grunwald
+ * \date   18.09.2017
+ * \brief
+ *
+ * \copyright
+ * Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+#pragma once
+
+#include "../mpProperty.h"
+
+namespace MaterialPropertyLib
+{
+class Medium;
+class Phase;
+class Component;
+/**
+ * \class IdealGasLaw
+ */
+class IdealGasLaw final : public Property
+{
+private:
+    /// A pointer to the phase object.
+    Phase* _phase;
+    Component* _component;
+
+public:
+    /// Constructor passing a pointer to the medium.
+    IdealGasLaw(Medium*);
```

Comment:
According to your comments in the cpp-file this constructor throws an error. For what reason do you need the constructor?